pkg/madmin/user-commands.go

```diff
@@ -42,6 +42,8 @@ type BucketUsageInfo struct {
 	Size    uint64        `json:"size"`
 	Created time.Time     `json:"created"`
 	Access  AccountAccess `json:"access"`
+
+	PrefixesUsage map[string]uint64 `json:"prefixesUsage"`
```

Comment:
This could get *huge*. I would prefer requests were on a per bucket level.

It is definitely not an API I would commit to long term.

Reply:
> This could get _huge_. I would prefer requests were on a per bucket level.

Sorry, just saw your comment.

Yeah, we can even do per bucket, per prefix as well. I mean we can oblige the user to send a prefix instead of listing all of them.